README.md

```diff
@@ -37,6 +37,11 @@ A memory provider that provides memory from an operating system.
 
 ## Memory pool managers
 
+### proxy_pool (part of libumf)
+
+This memory pool is distributed as part of libumf. It forwards all requests to the underlying
+memory provider.
```

Comment:
Could you add an info here what functions are not supported and why? This is not obvious without knowledge of memory provider API.

Reply:
Done.